### PR TITLE
Remove Travis CI stages which are duplicated

### PR DESCRIPTION
**Pull request by hickeyma**
The stages are now supported as GH workflow steps and therefore only duplication in CI.

Related #613

### CODE REVIEW

**Comment by hickeyma** [comment]
@saschagrunert @mikebrow This is ready for review.

**Comment by hickeyma** [comment]
Sorry but did last commit with wrong details. I amended the commit with correct details then. However, bot is not picking up new details. Is there a way to reset it?

**Comment by mikebrow** [comment]
squash the tidy commit into the good commit (the pick).. then edit the commit header of the pick making it one line again.. then push -f

**Comment by hickeyma** [comment]
Thanks @mikebrow, the squash did it.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -2,19 +2,19 @@ name: build
...
       matrix:
-        os: [ubuntu-18.04]  # TODO: pass linters on 'macos-10.15' 'windows-2019'
+        os: [ubuntu-18.04, macos-10.15]  # TODO: pass linters on 'windows-2019'
 
     steps:
       - name: Install Go
         uses: actions/setup-go@v1
         with:
-          go-version: '1.13.11'
+          go-version: '1.14'
```
Comment (mikebrow) [inline .github/workflows/build.yml:17]: a lot of groups are skipping 1.14 due to known issues with EINTR https://golang.org/doc/go1.14#runtime

Comment (mikebrow) [reply .github/workflows/build.yml:17]: did kubernetes move up to 1.14?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -35,20 +35,19 @@ jobs:
         working-directory: src/github.com/kubernetes-sigs/cri-tools
 
   releases:
-    name: releases
+    name: releases / on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 10
+    timeout-minutes: 15
```
Comment (mikebrow) [inline .github/workflows/build.yml:40]: how close was it that we had to go to 15?

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -2,19 +2,19 @@ name: build
 on: [push, pull_request]
```
Comment (mikebrow) [inline .github/workflows/build.yml:2]: suggest build and verify

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -2,19 +2,19 @@ name: build
...
 jobs:
   linters:
-    name: linters
+    name: linters / on ${{ matrix.os }}
```
Comment (mikebrow) [inline .github/workflows/build.yml:5]: suggest lint and vendor check

File: .travis.yml [hunk 5]
```diff
@@ -10,68 +10,22 @@ env:
   - GOSUMDB=https://sum.golang.org
 
 os:
-  - linux
   - linux-ppc64le
```
Comment (mikebrow) [inline .travis.yml:13]: suggest a TODO here to remove travis yml if/when github actions supports the remaining platforms...

File: .github/workflows/build.yml (hunk 1 above)
Comment (mikebrow) [reply .github/workflows/build.yml:17]: thx for the 1/1 discussion @hickeyma ...per discussion suggest following k8s model of using 1.13 for vendor and 1.13 and/or 1.14 for build/test/support

Comment (hickeyma) [reply .github/workflows/build.yml:17]: Yes, Kubernetes moved up in: https://github.com/kubernetes/kubernetes/pull/92438. Thanks for heads up on the issue in Go 1.14.

Thanks for the discussion @mikebrow, it was very enlightening. I have raised #639 to cover Go 1.14 support. In the meantime, I am going to revert to 1.13 for this PR.

File: .github/workflows/build.yml (hunk 2 above)
Comment (hickeyma) [reply .github/workflows/build.yml:40]: Raised to `20` to be sure. It was happening intermittently to Mac OS release build.

File: .github/workflows/build.yml (hunk 4 above)
Comment (hickeyma) [reply .github/workflows/build.yml:5]: Moved to #639

File: .github/workflows/build.yml (hunk 3 above)
Comment (hickeyma) [reply .github/workflows/build.yml:2]: Moved to #639

File: .travis.yml (hunk 5 above)
Comment (hickeyma) [reply .travis.yml:13]: done

**Comment by hickeyma** [comment]
Thanks for the review @mikebrow and especially the info on the known issue in Go 1.14. Updated and ready for review again.

File: .github/workflows/build.yml (hunk 4 above)
Comment (mikebrow) [reply .github/workflows/build.yml:5]: s/linters/lint and vendor check/
see line 34 below

**Comment by hickeyma** [comment]
Thanks for the review @mikebrow. Small nit updated.

**Comment by hickeyma** [comment]
Thanks for merging @saschagrunert

**Pull request merged**